package.json

```diff
@@ -27,8 +27,8 @@
   },
   "scripts": {
```

Comment:
remember to update the package version so new version of the package is pushed to npm